### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by Sepehrworklife** [Weblink-Co/schedulize#1] [General]
Always Smile :smile:

**Comment by Sepehrworklife** [comment]
@amiradiib Hello. If you have any free time I'll be happy if you contribute on this project.

**Reply by amiradiib** [reply]
Hi @Sepehrworklife Thanks
sure I'd be grateful to contribute.

**Comment by Sepehrworklife** [comment]
@erfanMirhoseini78 Welcome :heart:

**Reply by erfanMirhoseini78** [reply]
Thank you

**Comment by Sepehrworklife** [comment]
@erfanMirhoseini78 @amiradiib It will be good if you guys spend time learning the Vuejs.
Because this project is a monolith one, we will work with the Vuejs global.
In the future, I will add some issues if you want to contribute to any of them just let me know.

**Reply by erfanMirhoseini78** [reply]
why Vuejs?

**Reply by Sepehrworklife** [reply]
We have decided to keep our application as a monolith and not separate the frontend and backend. After considering htmx, we have concluded that vueJs is a better choice due to its extensive plugin library, strong community support, and ease of use. Rest assured, it is a straightforward technology to learn for those who are familiar with react.

**Reply by erfanMirhoseini78** [reply]
Thank you for your explanation

**Reply by Sepehrworklife** [reply]
It will be good practice for you guys to be familiar with monolith projects and use new stacks you haven't used yet.

**Reply by Sepehrworklife** [reply]
The pug template engine is another thing you should spend time for :laughing:

**Comment by Sepehrworklife** [comment]
@erfanMirhoseini78, please check out #4 and review the codes to understand the basic structure we are using.

**Reply by Sepehrworklife** [reply]
@amiradiib

**Reply by amiradiib** [reply]
@Sepehrworklife I also looked at the part you wanted me to review.

**Reply by erfanMirhoseini78** [reply]
@Sepehrworklife 

okay, I review the codes

**Comment by Sepehrworklife** [comment]
@erfanMirhoseini78 @amiradiib Sup guys, I did reinitialize the project with the NextJs (Turborepo).
if you want you can read about it. I hope you are happier about the decision :smile: 
I do create more and more issues in a time and like in the past, you can let me know to assign those to you.

**Reply by Sepehrworklife** [reply]
We are using `pnpm` instead of `npm` as a package manager. For running the development server you can use `pnpm dev`. for more information please see README.md file.

**Reply by amiradiib** [reply]
Thanks for informing us @Sepehrworklife 🙏🏼
